pkg/network/master/master.go

```diff
@@ -66,17 +66,22 @@ func Start(networkConfig osconfigapi.MasterNetworkConfig, networkClient networkc
 		panic("No ClusterNetworks set in networkConfig; should have been defaulted in if not configured")
 	}
 
+	var parsedClusterNetworkEntries []networkapi.ClusterNetworkEntry
+	for _, entry := range master.networkInfo.ClusterNetworks {
+		parsedClusterNetworkEntries = append(parsedClusterNetworkEntries, networkapi.ClusterNetworkEntry{CIDR: entry.ClusterCIDR.String(), HostSubnetLength: entry.HostSubnetLength})
+	}
```

Comment:
Another reason for converting master.networkInfo.ClusterNetworks to []networkapi.ClusterNetworkEntry